include/LLGL/Platform/Linux/LinuxNativeHandle.h

```diff
@@ -36,6 +39,59 @@ struct NativeHandle
     int             screen;
 };
 
+#ifdef LLGL_LINUX_ENABLE_WAYLAND
```

Comment:
Some minor nitpick: I started to rely much more on `#if` instead of `#ifdef` in case those macros are defined as `0`. CMake automatically defines them to `1` when you don't specify a value, so this will still work.